edpm_ansible/roles/edpm_ovn/tasks/configure.yml

```diff
@@ -45,3 +45,82 @@
+
+- name: Add OVS Manager
+  block:
+    - name: Check if OVS Manager already exists
+      ansible.builtin.shell: >
+        ovs-vsctl show | grep -q "Manager"
+      register: ovs_manager_configured
+      ignore_errors: yes
+    - name: Add OVS Manager if not exists
+      ansible.builtin.shell: >
+        ovs-vsctl --timeout=5 --id=@manager -- create Manager target=\"ptcp:6640:127.0.0.1\" -- add Open_vSwitch . manager_options @manager
+      when: ovs_manager_configured.rc == 1
+
+- name: Configure neutron configuration files
+  block:
+    - name: Ensure {{ edpm_ovn_neutron_config_dir }}/etc/neutron/plugins/networking-ovn exists
+      ansible.builtin.file:
+        path: "{{ edpm_ovn_neutron_config_dir }}/etc/neutron/plugins/networking-ovn"
+        state: directory
+        recurse: true
+
+    - name: Remove neutron_metadata_config container
+      ansible.builtin.shell: >
+        podman rm -f neutron_metadata_config || :
+
+    - name: Run neutron_metadata_agent container
+      ansible.builtin.shell: >
+          podman run --detach --name neutron_metadata_config quay.io/tripleomastercentos9/openstack-neutron-metadata-agent-ovn:current-tripleo sleep infinity
+      register: metadata_container_id
+
+    - name: register_metadata_id
+      ansible.builtin.debug:
+        msg: "{{ metadata_container_id }}"
+
+    - name: Copy initial neutron.conf files
+      ansible.builtin.shell: |
+        mount_dir=$(podman mount neutron_metadata_config)
+        cp -a ${mount_dir}/etc/neutron/neutron.conf {{ edpm_ovn_neutron_config_dir }}/etc/neutron/neutron.conf
+      failed_when: false
+      register: copy_config_files
+
+    - name: Copy initial rootwrap.conf files
+      ansible.builtin.shell: |
+        mount_dir=$(podman mount neutron_metadata_config)
+        cp -a ${mount_dir}/etc/neutron/rootwrap.conf {{ edpm_ovn_neutron_config_dir }}/etc/neutron/rootwrap.conf
+      failed_when: false
+      register: copy_config_files
+
+    - name: Copy initial neutron-metadata-ini.conf files
+      ansible.builtin.shell: |
+        mount_dir=$(podman mount neutron_metadata_config)
+        cp -a ${mount_dir}/etc/neutron/neutron_ovn_metadata_agent.ini {{ edpm_ovn_neutron_config_dir }}/etc/neutron/plugins/networking-ovn/networking-ovn-metadata-agent.ini
+      failed_when: false
+      register: copy_config_files
+
+    - name: Remove neutron_metadata_config container
+      ansible.builtin.shell: >
+        podman rm -f neutron_metadata_config || :
+
+    - name: Configure neutron.conf
+      config_template:
+        src: "{{ edpm_ovn_neutron_config_dir }}/etc/neutron/neutron.conf"
+        dest: "{{ edpm_ovn_neutron_config_dir }}/etc/neutron/neutron.conf"
+        config_type: ini
+        config_overrides: "{{ edpm_ovn_metadata_agent_neutron_config | combine(edpm_ovn_metadata_agent_neutron_config_overrides, recursive=True) }}"
+
+    - name: Configure rootwrap.conf
+      config_template:
```

Comment:
please, replace the task with its fqcn: `openstack.config_template.config_template`